src/test/rustdoc/primitive/primitive-generic-impl.rs

```diff
@@ -1,3 +1,7 @@
+#![feature(rustdoc_internals)]
+
+// FIXME: this test doesn't test anything
+
```

Comment:
Note that this file is included as part of another test, so the feature gate is invalid and the note is incorrect. But this confusing `include!` behavior is removed in a later commit.